### PR TITLE
Fix: #120. Add NDC profile.

### LINKED ISSUE #120: Define `ndc` application profile
## I expect

Every vocabulary has the following properties:

- ndc:keyConcept - an ascii 7bit string identifier that is unique in each repository and with regard to a specific NDC namespace; it is used to identify the vocabulary to be published as REST API, eg. `/vocabularies/v1/vocabularies/{keyConcept}`

- ndc:controlledVocabularyURI - a property to correlate a vocabulary with the REST API server url. 

## Notes

Property names may be changed

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

adds the NDC profile contributed by @giorgialodi 

Fixes #120

### CODE REVIEW

**Comment by giorgialodi** [comment]
@ioggstream dovrebbe essere ok

**Review by giorgialodi — COMMENTED** [review]
I think it is wise if we change the URI... let's use the following https://w3id.org/italia/onto/NDC/ I will change it and re-send to you

File: Ontologie/NDC/latest/NDC-AP_IT.ttl [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+                                                                                   dcatapit:Agent ;
+                                                                          dct:identifier "cnr - Z6HZEH - STLab" ;
+                                                                          foaf:name "Institute of Cognitive Sciences and Technologies of the Italian Research Council (CNR) - Semantic Technology Laboratory (STLab)"@en ,
+                                                                                    "Istituto di Scienze e Tecnologie della Cognizione del CNR - Semantic Technology Lab (STLab)"@it .
+
+
+###  https://w3id.org/italia/data/project/OntoPiA
+<https://w3id.org/italia/data/project/OntoPiA> rdf:type owl:NamedIndividual ,
+                                                        admsapit:Project ;
+                                               :name "OntoPiA - la rete di ontologie della pubblica amministrazione italiana"@it .
```
Comment (ronytw) [inline Ontologie/NDC/latest/NDC-AP_IT.ttl:303]:
```suggestion
                                               foaf:name "OntoPiA - la rete di ontologie della pubblica amministrazione italiana"@it .
```

Comment (giorgialodi) [reply Ontologie/NDC/latest/NDC-AP_IT.ttl:303]: No @ronytw It has been already fixed and it is l0:name non foaf:name.

File: Ontologie/NDC/latest/NDC-AP_IT.ttl [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+
+#################################################################
+#    Data properties
+#################################################################
+
+###  https://w3id.org/italia/onto/ndc-profile/keyConcept
+:keyConcept rdf:type owl:DatatypeProperty ;
+            rdfs:domain dcatapit:Dataset ;
+            rdfs:range rdfs:Literal ;
+            rdfs:comment "Questa proprietà consente di specifcare un concetto chiave rappresentato dal dataset. Nell'ambito del profilo applicativo per il catalogo nazionale dati, il concetto chiave è associato ai vocabolari controllati, che sono dataset, per indicare il concetto chiave che rappresentano come vocabolari di riferimento. Esempio: il dataset https://w3id.org/italia/controlled-vocabulary/classifications-for-public-services/authentication-type ha come concetto chiave \"authentication type\" o \"tipo di autenticazione\""@it ,
```
Comment (ioggstream) [inline Ontologie/NDC/latest/NDC-AP_IT.ttl:162]:
```suggestion
:keyConcept a owl:DatatypeProperty ;
            rdfs:domain dcatapit:Dataset ;
            rdfs:range rdfs:Literal ;
            rdfs:comment "Questa proprietà consente di specifcare un concetto chiave rappresentato dal dataset. Nell'ambito del profilo applicativo per il catalogo nazionale dati, il concetto chiave è associato ai vocabolari controllati, che sono dataset, per indicare il concetto chiave che rappresentano come vocabolari di riferimento. Esempio: il dataset https://w3id.org/italia/controlled-vocabulary/classifications-for-public-services/authentication-type ha come concetto chiave \"authentication type\" o \"tipo di autenticazione\""@it ,
```

File: Ontologie/NDC/latest/NDC-AP_IT.ttl [hunk 3]
```diff
@@ -0,0 +1,326 @@
+@prefix : <https://w3id.org/italia/onto/ndc-profile/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
```
Comment (giorgialodi) [inline Ontologie/NDC/latest/NDC-AP_IT.ttl:6]:
```suggestion 
pippo
```

**Comment by giorgialodi** [comment]
@ioggstream @ronytw revised the file of this pull request. See third commit.

**Comment by ioggstream** [comment]
minor typos

* Sort languages. Add "for semantic interoperability".

**Review by mfortini — APPROVED** [review]

**Pull request merged**